### PR TITLE
Removed validation for test persons

### PR DESCRIPTION
**Pull request by HermannBjorgvin**
Since real persons with 0 or 1 as 7th char exist as edge cases I am removing validation for test persons.

And since the person with a KT like that can be valid then test persons will also always be valid, so no need to add an option to allow the test dataset

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -3,55 +3,38 @@
...
   generatePerson,
   generateCompany,
   generateTemporary,
 } from "./generation";
-import { KennitalaInfo, ValidationOptions } from "./types";
+import { KennitalaInfo } from "./types";
 
-export const isValid = (
-  kennitala: string,
-  options?: ValidationOptions
```
Comment (eirikurn) [inline src/index.ts:22]: This is a breaking change to the TypeScript definition. Wonder if we should just deprecate and ignore `options` or bump to v4?

Comment (HermannBjorgvin) [reply src/index.ts:22]: Yeah might be good to patch v3 since it's been available for a while now. Will change this to a patch vs major and v4 can be a separate PR

Comment (HermannBjorgvin) [reply src/index.ts:22]: Opened another pr #45 

Will publish 3.0.4 with the Misc suggestions PR and that one.
